packages/storybook/src/stories/grid-toolbar.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { XGrid } from '@material-ui/x-grid';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withA11y } from '@storybook/addon-a11y';
+import '../style/grid-stories.css';
+import { useData } from '../hooks/useData';
+
+export default {
+  title: 'X-Grid Tests/Toolbar',
+  component: XGrid,
+  decorators: [withKnobs, withA11y],
+  parameters: {
+    options: { selectedPanel: 'storybook/storysource/panel' },
+    docs: {
+      page: null,
+    },
+  },
+};
+export const DensityPicker = () => {
```

Comment:
Do we need this story? Maybe we should have 2 stories with `size="small"` and `size="large"`?

Reply:
In the documentation, I assume we will only display one variant of the size of the grid, which won't be enough for visual regression tests.

Reply:
Makes sense. I'll add 2 stories and remove the default case.